plugin-flex-ts-template-v2/src/feature-library/dispositions/flex-hooks/actions/CompleteTask.ts

```diff
@@ -26,19 +36,23 @@ export const actionHook = function setDispositionBeforeCompleteTask(flex: typeof
     if (!payload.task?.taskSid) {
       return;
     }
-
-    const numDispositions = getDispositionsForQueue(payload.task?.queueSid ?? '').length;
+    let queueName = '';
+    const queue = await FlexHelper.getQueue(queueName);
+    if (queue) queueName = queue?.queue_name;
+    const numDispositions = getDispositionsForQueue(queueName ?? '').length;
```

Comment:
can we modify these functions to support looking up by queue name and queue sid - if we don't also look up by queue sid - it will break any existing configurations

Reply:
Do you expect customer to deploy the updated Template plugin code but NOT the updated config?

Reply:
No but they will have to merge it

Reply:
Ok, done!